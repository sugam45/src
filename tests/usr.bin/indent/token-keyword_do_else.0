/* $NetBSD: token-keyword_do_else.0,v 1.1 2021/03/12 00:13:06 rillig Exp $ */
/* $FreeBSD$ */

/*
 * Tests for the keyword 'do' or 'else'.  These two keywords are followed by
 * a space.  In contrast to 'for', 'if' and 'while', the space is not
 * followed by a parenthesized expression.
 */

/* TODO: Add some code to be formatted. */
