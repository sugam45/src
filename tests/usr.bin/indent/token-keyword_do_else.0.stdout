/* $NetBSD: token-keyword_do_else.0.stdout,v 1.2 2021/03/14 01:34:13 rillig Exp $ */
/* $FreeBSD$ */

/*
 * Tests for the keyword 'do' or 'else'.  These two keywords are followed by a
 * space.  In contrast to 'for', 'if' and 'while', the space is not followed
 * by a parenthesized expression.
 */

/* TODO: Add some code to be formatted. */
